client/payment-details/dispute-details/dispute-notice.tsx

```diff
@@ -52,6 +54,16 @@ const DisputeNotice: React.FC< DisputeNoticeProps > = ( {
 		);
 		learnMoreDocsUrl =
 			'https://woocommerce.com/document/woopayments/fraud-and-disputes/managing-disputes/#inquiries';
+
+		if ( paymentMethod === 'klarna' ) {
+			noticeText = __(
+				'Klarna inquiries may mean that the customer is trying to return their item(s). ' +
+					'<a>Please see this document for more information</a>',
```

Comment:
I've made a tiny change, removed the `.` at the end of the link, to make it consistent with other dispute notices.

<img width="970" alt="image" src="https://github.com/user-attachments/assets/5820dd3a-1b66-4d1d-a8ff-e36885102a36" />


<img width="963" alt="image" src="https://github.com/user-attachments/assets/da92642b-25b1-4fbf-bab2-e7b8a1ddea13" />

Update after 28129244dd5e6403f9601ff16ca3c2fd1bb656b6:

<img width="958" alt="image" src="https://github.com/user-attachments/assets/f9b4647d-cbb3-4589-897e-3a51376e09df" />

Reply:
Thank you!